packages/kit/src/exports/vite/index.js

```diff
@@ -685,6 +705,18 @@ async function kit({ svelte_config }) {
 			}
 		},
 
+		renderChunk(code, chunk) {
+			if (code.includes('__SVELTEKIT_TRACK__')) {
+				return {
+					code: code.replace(/__SVELTEKIT_TRACK__\('(.+?)'\)/g, (_, label) => {
```

Comment:
it could help to add a comment saying what the code is that's being output and what it does

Reply:
feels kinda duplicative with the code